File: HW01/PYTHON/project/TableTennisPaddle.py
from .IndoorItem import IndoorItem

class TableTennisPaddle(IndoorItem):
    def __init__(self, price, style):
        super().__init__(price)
        self.style = style
    
    def display(self):
        print(f"${self.price} - This table tennis paddle is for {self.style}", end="")

File: HW01/CPP/src/TableTennisPaddle.cpp
#include "../include/TableTennisPaddle.h"
#include <iostream>
#include <string>
using namespace std;

TableTennisPaddle::TableTennisPaddle(string style, double price) : IndoorItem(price){
    this->style = style;
    this->price = price;
}

void TableTennisPaddle::display() {
    cout << "$" << this->price << " - This table tennis paddle is for " << this->style << endl;
}